include/boost/unordered/detail/foa/node_handle.hpp

```diff
@@ -240,20 +240,8 @@ struct node_handle_access
   template <class TypePolicy, class Allocator>
   using node_type = node_handle_base<TypePolicy, Allocator>;
 
-#if BOOST_CLANG_VERSION < 190000
```

Comment:
Note: this section was added by me to workaround the linked clang bug. You'll need to add it back manually since otherwise builds fail, like for example the failing macos CI build. Ideally in a few more releases this will age out and we can get rid of it.